docs/serving/knative-kubernetes-services.md

```diff
@@ -33,41 +44,40 @@ networking-istio         1         1         1            1           1h
 webhook                  1         1         1            1           1h
 ```
 
-## Service: activator
+These serivces and deployments are installed by the `serving.yaml` file during install. The next section describes the function of these components.
 
-The responsibilities of the activator are:
+## Components
 
-- Receiving & buffering requests for inactive Revisions.
-- Reporting metrics to the autoscaler.
-- Retrying requests to a Revision after the autoscaler scales such Revision
-  based on the reported metrics.
+### Service: activator
 
-## Service: autoscaler
+The activator is responsible for receiving & buffering requests for inactive revisions and reporting metrics to the autoscaler. In addition, it retries requests to a revision after the autoscaler scales the revision based on the reported metrics.
 
-The autoscaler receives request metrics and adjusts the number of pods required
-to handle the load of traffic.
+### Service: autoscaler
 
-## Service: controller
+The autoscaler receives request metrics and adjusts the number of pods required to handle the load of traffic.
+
+### Service: controller
 
 The controller service reconciles all the public knative objects and autoscaling
-CRDs. When a user applies a knative service to the kubernetes api, this creates
-the config and route. It will convert config into revisions. It will convert
-Revision into Deployment and KPA.
+CRDs. When a user applies a Knative service to the Kubernetes API, this creates
+the configuration and route. It will convert the configuration into revisions, and the revisions into Deployment and Knative Pod Autoscaling (KPA).
```

Comment:
```suggestion
the configuration and route. It will convert the configuration into revisions and the revisions into deployments and Knative Pod Autoscalers (KPAs).
```